src/components/color-picker/color-picker.tsx

```diff
@@ -974,10 +974,7 @@ export class ColorPicker
   }
 
   private renderOpacitySection(): VNode {
-    const { color, previousColor } = this;
-    const sliderOpacity = alphaToOpacity((color ? color : previousColor).alpha()); // slider keeps previous alpha when null
-    const inputOpacity = color ? alphaToOpacity(color.alpha()).toString() : undefined;
```

Comment:
Why was this changed to match the sliderOpacity value? I assume there was a reason they were different in the first place, although I have no idea what it might be lol

Reply:
the only change here was to make the code more readable. using "this." rather than destructuring at the top of the page and there is no need for "inputOpacity" as this.color is a required prop with a fallback.